onnxruntime/core/session/inference_session.cc

```diff
@@ -924,7 +921,7 @@ common::Status InferenceSession::TransformGraph(onnxruntime::Graph& graph,
   // 4. insert copy nodes
   // 5. insert cast nodes.
 
-  // first apply global(execution provider independent),  level 1(default/system/basic) graph to graph optimizations
+  // first apply global(execution provider independent), level 1(default/system/basic) graph to graph optimizations
```

Comment:
I was confused by this comment. the next line applies Level1 transformers only (not TransformerLevel::Default ones).
how about something like this: `first apply execution provider independent, level 1 (basic) graph optimizations`

Reply:
I think it's confusing as there aren't actually any 'default' level transformers. In fact, 'default' is actually equivalent to 'none'

https://github.com/microsoft/onnxruntime/blob/006597b9b8d581c4db5e4294863a33c65cb49fb5/onnxruntime/core/session/abi_session_options.cc#L125-L127

I'll update the comment.